examples/boids/main.rs

```diff
@@ -116,9 +116,7 @@ impl framework::Example for Example {
             rasterization_state: Some(wgpu::RasterizationStateDescriptor {
                 front_face: wgpu::FrontFace::Ccw,
                 cull_mode: wgpu::CullMode::None,
-                depth_bias: 0,
-                depth_bias_slope_scale: 0.0,
-                depth_bias_clamp: 0.0,
+                ..Default::default()
```

Comment:
I think that using the default when initializing this descriptor is a good idea, since setting `depth_bias` to 0 or `depth_clamp` to `false` isn't very relevant to most of these examples.

Reply:
yep, that's great!